package.json

```diff
@@ -75,6 +75,7 @@
   "scripts": {
     "build": "gulp build",
     "lint": "gulp lint sass-lint",
+    "package": "cd addon && npm run package",
```

Comment:
hmm, yeah I guess this assumes that `npm install` has already been run. What do you recommend for this case?